### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by Shaniko**
### What changed
2 icons (trash3 and email)
...

### Why it changed
ux request
...

---

- [ ] Change is tested
- [ ] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by argshook** [comment]
@milkyfruit pretty please

**Comment by argshook** [comment]
@wuwa please check, @milkyfruit is on vacation this week

**Comment by wuwa** [comment]
@argshook checked it's fine

File: src/Icons/raw/warning.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (argshook) [inline src/Icons/raw/warning.svg]: @Shaniko uppercase `Warning` please

**Pull request merged**